partiql-ast/src/main/resources/partiql_ast.ion

```diff
@@ -785,23 +788,27 @@ returning::{
 // `( CONSTRAINT <column_constraint_name> )?  <column_constraint_def>`
 table_definition::{
   columns: list::[column],
+  // table level constraints
+  constraints: list::[constraint],
   _: [
     column::{
-      name:        string,
+      name:        '.identifier.symbol',
       type:        '.type',
       constraints: list::[constraint],
-      _: [
-        // TODO improve modeling language to avoid these wrapped unions
-        // Also, prefer not to nest more than twice
-        constraint::{
-          name: optional::string,
-          body: [
-            nullable::{},
-            not_null::{},
-            check::{ expr: expr },
-          ],
-        },
-      ],
+    }
+  ],
+}
+
+constraint::{
+  name: optional::string,
+  body: [
```

Comment:
(naming) not sure if `body` is a good name. Perhaps could use `type` or something else?

Reply:
I do not like the name either. 

To address this and other naming suggestion in the comments: 

The name are reused from the previous implementation. I am more than happy to change those: 

"column" -> "attribute" 
"body" -> "definition"